src/Compilers/CSharp/Test/Syntax/Diagnostics/DiagnosticTest.cs

```diff
@@ -1867,11 +1868,197 @@ public static void Main()
                 // (13,25): warning CS1072: Expected identifier or numeric literal.
                 // #pragma warning restore "CS0168
                 Diagnostic(ErrorCode.WRN_IdentifierOrNumericLiteralExpected, @"""CS0168").WithLocation(13, 25),
-                // (14,17): warning CS1634: Expected disable or restore
+                // (14,17): warning CS1634: Expected disable, restore, enable or safeonly
                 // #pragma warning blah
                 Diagnostic(ErrorCode.WRN_IllegalPPWarning, "blah").WithLocation(14, 17));
         }
 
+        [Fact]
+        public void PragmaWarning_EnableSafeOnlyNullable()
+        {
+            var text = @"
+#pragma warning disable nullable
+#pragma warning enable nullable
+#pragma warning restore nullable
+#pragma warning safeonly nullable
+
+#pragma warning disable nullable, 1695
+#pragma warning enable nullable, 1695
+#pragma warning restore nullable, 1695
+#pragma warning safeonly nullable, 1695
+
+#pragma warning enable 
+#pragma warning safeonly
+#pragma warning enable 1695
+#pragma warning safeonly 1695
+";
+            CreateCompilation(text, parseOptions: TestOptions.Regular7_3).VerifyDiagnostics(
+                // (3,17): warning CS1658: Feature 'warning action enable or safeonly' is not available in C# 7.3. Please use language version 8.0 or greater.. See also error CS8370.
```

Comment:
>See also error CS8370. [](start = 160, length = 22)

What does this mean?

Reply:
> What does this mean?

I assume this means that that error was changed to a warning and documentation for it should be looked at for more information about why the warning is reported. In this case it is a language version mismatch error converted to a warning

---
In reply to: [245846859](https://github.com/dotnet/roslyn/pull/32208#discussion_r245846859) [](ancestors = 245846859)